.gitignore

```diff
@@ -3,6 +3,7 @@ build/*
 ansible/inventory
 ansible/**/__pycache__
 yarn-error.log
+.idea
```

Comment:
You should probably add this to your global `.gitignore` instead

Reply:
Will do 👍🏼

Reply:
Done.